Tasks/QuickPerfTest/task.json

```diff
@@ -3,7 +3,7 @@
   "name": "QuickPerfTest",
   "friendlyName": "Cloud-based Web Performance Test",
```

Comment:
Can you please add Readme for the same if you are touching it. It would be great to understand what's this task for and usage